src/map/entities/battleentity.cpp

```diff
@@ -1077,7 +1096,25 @@ uint16 CBattleEntity::ACC(uint8 attackNumber, uint16 offsetAccuracy)
                 ACC += this->PMaster->getMod(Mod::TANDEM_STRIKE_POWER);
             }
         }
+        if (this->objtype == TYPE_PET)
+        {
+            auto getEcoStrBonusFunc = lua["utils"]["getEcosystemStrengthBonus"];
 
+            if (getEcoStrBonusFunc.valid())
+            {
+                CBattleEntity* thisTarget = nullptr;
+                if (this->PAI->IsEngaged())
+                {
+                    thisTarget = this->GetBattleTarget();
+                }
+                if (
+                    thisTarget != nullptr &&
```

Comment:
While this is the formatting we use in Lua, we don't use this in C++-land, these conditions can go on single lines, and there can be a break in between the two if statements here